components/theme/app/AppLayout.vue

```diff
@@ -15,8 +16,9 @@
         <slot />
       </div>
     </div>
+    <AppFooter :links="footerLinks" :class="showCookieBanner ? 'pb-16 md:pb-12' : ''" />
 
-    <AppFooter :links="footerLinks" />
+    <CookieBanner class="w-full fixed bottom-0 left-0 z-40" @cookie-banner="showCookieBanner = false" />
```

Comment:
Really minor comment, but I feel like doing `inset-x-0` is more secure and simpler than `w-full left-0` 🙂

Reply:
👍